### ISSUE TITLE
Need a way to make "click to zoom out" behaviour optional with brushing/zooming

### ISSUE THREAD

**Issue by wlach** [metricsgraphics/metrics-graphics#886]
In Perfherder's use case, one graph (the top one) allows the user to click on it to unset any zooming. However, for the lower one, this is not the case -- clicking on the graph has no effect. Right now in metricsgraphics, clicking on a zoomed-in graph always zooms out.

I'm open to being convinced of another solution, but it seems to me like the simplest solution is to add a `click_to_zoom_out` boolean parameter to data graphic to make this behaviour optional. @cnwangjie what do you think?

**Comment by cnwangjie** [comment]
Yes, I think it is indeed the simplest solution in this case. I've submitted a PR to add `click_to_zoom_out` option. (#887)
